specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/definitions.json

```diff
@@ -1939,6 +1939,27 @@
       ],
       "description": "Authorization header information."
     },
+    "BackendManagedIdentityCredentials": {
```

Comment:
>BackendManagedIdentityCredentials [](http://example.com/codeflow?start=5&length=33)

Can you refer to [https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json)

and see if you can refer any of these , instead of a new definition here ?

Reply:
These are not the same type of object. The existing contracts describe the service's assigned managed identities (system or user assigned). Whereas the new contract describes the managed identity used to target APIM backends. The are related but not the same thing.